e2e_samples/parking_sensors_synapse/README.md

```diff
@@ -51,7 +51,7 @@ Sample PowerBI report
 
 The following shows the overall CI/CD process end to end.
 
-![CI/CD](docs/images/CI_CD_process.PNG?raw=true "CI/CD")
+![CI/CD](docs/images/CI_CD_process.png?raw=true "CI/CD")
```

Comment:
Link broken, the file name is "CI_CD_process.PNG".

Reply:
Ah I must have gone a bit overboard and change too many file names because of the link checker :D I think I'll just change the file name to lowercase to make it consistent.

Reply:
Fixed :)